src/custom-hooks/useFetch.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../types/Person';
+
+function wait(delay: number) {
+  return new Promise(resolve => setTimeout(resolve, delay));
+}
+
+export function useFetch(url: string) {
```

Comment:
```suggestion
export function userFetchPeople() {
```
this hook is tied to people-related flow, so there's no need to pass url as a param — we always retrieve users